src/components/atoms/Chip/Chip.tsx

```diff
@@ -1,24 +1,25 @@
 import clsx from 'clsx';
+import { colorStyle, sizeStyle } from './constants';
+import { ChipColors, ChipSizes } from './types';
 
-export const Chip = ({ label }: { label: 'Borrow' | 'Lend' }) => {
+export const Chip = ({
+    color = ChipColors.Gray,
+    size = ChipSizes.md,
+    label = 'Label',
+}: {
+    color?: ChipColors;
+    size?: ChipSizes;
+    label: string;
```

Comment:
when label is not optional, providing the default value should not be doing anything?

Reply:
Changing `label` to be mandatory and removing the default